sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -281,6 +281,45 @@ public virtual async Task<Response> RevokeTokensAsync(CommunicationUserIdentifie
             }
         }
 
+        /// <summary>Exchange an AAD access token of a Teams user for a new ACS access token.</summary>
+        /// <param name="token">Azure Active Directory access token to acquire a new ACS access token.</param>
+        /// <param name="cancellationToken">The cancellation token to use.</param>
+        /// <exception cref="RequestFailedException">The server returned an error.</exception>
+        public virtual Response<AccessToken> ExchangeAccessToken(string token, CancellationToken cancellationToken = default)
```

Comment:
Token or AccessToken in the entire context has been implying the ACS tokens.. so I think it will make sense for us to be more explicit and call it either ExchangeTeamsToken or ExchangeAADToken. Having AAD in the name might again be confusing because AAD can imply in the Azure context as well so let's keep the former (ExchangeTeamsToken) and that also coincides with the API name

Reply:
I'll update the operation name to ExchangeTeamsToken as well as the parameter name to teamsToken, so it's clear.